src/API.jl

```diff
@@ -905,10 +905,8 @@ end
 
 _do_auto_precompile() = parse(Int, get(ENV, "JULIA_PKG_PRECOMPILE_AUTO", "0")) == 1
 
-precompile() = precompile(Context())
-function precompile(ctx::Context)
-    printpkgstyle(ctx, :Precompiling, "project...")
-    
+precompile(auto::Bool=false) = precompile(Context(), auto)
```

Comment:
Maybe make this a kwarg? And maybe a more "scary internal name" like `internal_clall` or something?

Reply:
Done